prow/prowjobs/google/exposure-notifications-server/en-server.yaml

```diff
@@ -35,7 +35,7 @@ presubmits:
         secret:
           secretName: gcs-access-service-account
 periodics:
-  - cron: "0 8 * * *"  # Run at 00:00 PST
+  - cron: "* */3 * * *"  # Run every 3 hours
```

Comment:
Should this be `0 */3 * * *`? I don't know cron super well but I am not sure if the * in minutes would mean every minute every 3rd hour

Reply:
You are right. It turns out that I'm even less familiar with cron than you are :) 
Fixed